shared/middleware/helpers/null_pub_sub_api.rb

```diff
@@ -0,0 +1,11 @@
+# Interface to fake interactions with a PubSub system, like Pusher
+# Should respond to same messages as PusherApi
+
+class NullPubSubApi
+  def self.configure_keys
```

Comment:
The name is a bit implementation specific.  I think init would be a better name, but I think you can probably avoid it entirely by having the implementation init itself in the constructor or in the publish method.

If you decided to keep it, please add documentation explaining when the calller should call this method.